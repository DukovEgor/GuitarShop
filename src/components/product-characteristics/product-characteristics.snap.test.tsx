import { render } from '@testing-library/react';
import ProductCharacteristics from './product-characteristics';

describe('Component: ProductCharacteristics', () => {
  test('should render correctly', () => {
    const { container } = render(<ProductCharacteristics vendorCode={'123f'} type={'electric'} description={'description'} stringCount={7} />);

    expect(container).toMatchSnapshot();
  });
});
