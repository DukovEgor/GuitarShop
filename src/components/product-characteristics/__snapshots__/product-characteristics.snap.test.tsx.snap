// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: ProductCharacteristics should render correctly 1`] = `
<div>
  <table
    class="tabs__table"
  >
    <tbody>
      <tr
        class="tabs__table-row"
      >
        <td
          class="tabs__title"
        >
          Артикул:
        </td>
        <td
          class="tabs__value"
        >
          123f
        </td>
      </tr>
      <tr
        class="tabs__table-row"
      >
        <td
          class="tabs__title"
        >
          Тип:
        </td>
        <td
          class="tabs__value"
        >
          Электрогитара
        </td>
      </tr>
      <tr
        class="tabs__table-row"
      >
        <td
          class="tabs__title"
        >
          Количество струн:
        </td>
        <td
          class="tabs__value"
        >
          7
           струнная
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
